src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTiming.java

```diff
@@ -308,7 +298,7 @@ public static GenericStatus computeChunkStatus2(@Nonnull WorkflowRun run,
             if(afterError != null) {
                 Throwable rootCause = afterError.getError();
                 if (rootCause instanceof FlowInterruptedException) {
-                    return GenericStatus.ABORTED;
+                    return GenericStatus.fromResult(((FlowInterruptedException) rootCause).getResult());
```

Comment:
@svanoort I have an additional change that modifies the return statement at the end of this method to return `GenericStatus.SUCCESS` despite the run result. In my local environment this really improves the situation described in [JENKINS-39203](https://issues.jenkins-ci.org/browse/JENKINS-39203).

Reply:
I haven't tested this locally, but I assume it would still return `ABORTED` for the aborted input example in https://github.com/jenkinsci/pipeline-graph-analysis-plugin/pull/19 ?

Reply:
@ghenzler Yes it does. I just tested this with the snippet you provided in  #19:
![image](https://user-images.githubusercontent.com/551572/50251012-5acf7080-0397-11e9-95dd-86c92879dc39.png)

Reply:
Implementation of aborting the input step is here: [InputStepExecution.java#L229](https://github.com/jenkinsci/pipeline-input-step-plugin/blob/41d514d266722433181e860d8cf90d56bca5046e/src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java#L229). A new `FlowInterruptedException` is created with a `Result.ABORTED` result.